cmake/IgnUtils.cmake

```diff
@@ -886,6 +887,10 @@ endmacro()
 #                    function. The target name will always be
 #                    ${PROJECT_LIBRARY_TARGET_NAME}.
 #
+# [LEGACY_PROJECT_PREFIX]: Optional. The variable that follows this argument will be
+#                          used as a prefix for the legacy cmake config variables
+#                          <prefix>_LIBRARIES and <prefix>_INCLUDE_DIRS.
```

Comment:
Can this be moved to be above the documentation for `GET_TARGET_NAME`, since `LEGACY_PROJECT_NAME` comes before `GET_TARGET_NAME` in the `ign_create_core_library` call above?

Reply:
I reordered the arguments in the example call above to match this order